src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -495,6 +503,29 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                       onChange={(event: React.ChangeEvent<HTMLInputElement>) => setRows(event.target.value)}
                       required
                     />
+                  ) : renderingType === 'toggle' ? (
+                    <div>
+                      <TextInput
+                        id="lableTrue"
+                        labelText={t('Label true', 'Label true')}
```

Comment:
The first param passed into the `t` function is just the key and the second param is he default string to be shown
```suggestion
                        labelText={t('labelTrue', 'Label true')}
```